### PR TITLE
Show the instructors block in memberships and use it in membership template

### PR DESCRIPTION
**Pull request by alaa-alshamy**
## Description

My changes are add the instructors block in the memberships and add the block to membership template also

and there's another pull request to complete the work in the core repo:
https://github.com/gocodebox/lifterlms/pull/1466

## How has this been tested?

Manually

## Types of changes

Bug fix

## Checklist:

- [x] My code has been tested.
- [ ] My code passes all existing automated tests. 
- [x] My code follows the LifterLMS Coding & Documentation Standards.

### CODE REVIEW

**Comment by thomasplevy** [comment]
@alaa-alshamy Thanks for this,

We'll need documentation updates per our documentation standards. I only ask since you've contributed before and I'm pretty certain I've noted this to you in the past (if I'm wrong, my apologies).

If you want to update the docblocks please do. I'll get to it after the holidays if you don't update the PR before then.

Thanks!

**Comment by alaa-alshamy** [comment]
@thomasplevy Yes i think you did, sorry my bad but i am still new with contributing so some patience please :) 

Check it now please and let me know if i missed something

File: includes/class-llms-blocks-post-types.php [hunk 1]
```diff
@@ -5,7 +5,8 @@
  * @package  LifterLMS_Blocks/Main
  *
  * @since 1.0.0
- * @version 1.7.0
+ * @since [version] Add instructors block to membership editor template.
```
Comment (thomasplevy) [inline includes/class-llms-blocks-post-types.php]:
```suggestion
```

File: includes/class-llms-blocks-post-types.php [hunk 2]
```diff
@@ -17,6 +18,7 @@
  * @since 1.5.2 Only `show_in_rest` for authenticated users with the `lifterls_instructor` capability.
  * @since 1.7.0 Add membership categories and tags to WordPress REST API.
  *              Add membership post type editor template.
+ * @since [version] Add instructors block to membership editor template.
```
Comment (thomasplevy) [inline includes/class-llms-blocks-post-types.php]:
```suggestion
```

**Review by thomasplevy — APPROVED** [review]

**Comment by thomasplevy** [comment]
thanks @alaa-alshamy !

**Pull request merged**